### PR TITLE
Standardize event data for borrowing event 

### LINKED ISSUE #240: Standardize event data for borrowing event
1.  Read our [Contributor Guidelines](https://github.com/CarmineOptions/derisk-research/blob/master/CONTRIBUTING.md) and [README](https://github.com/CarmineOptions/derisk-research/blob/master/README.md).
2. Parse data from event which fetched on-chain to humman-readable format, for example, now we have `user = data[0]` and we want to have `user = data.user`
 - In method [parse_borrowing_event](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/handler_tools/data_parser/zklend.py#L28)
 - Event data structure - [event data](https://starkscan.co/event/0x076b1615750528635cf0b63ca80986b185acbd20fa37f0f2b5368a4f743931f8_3)
3. Create Serializers model using pydantic in [serializers.py](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/handler_tools/data_parser/serializers.py)
4. In method [process_borrowing_event](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/handlers/loan_states/zklend/events.py#L281)
 substitute fetching data by index `Example: user= data[0]` to `data.user` using serializer class

### PR DESCRIPTION
**Pull request by ikemHood**
May fix #240 

- added BorrowingEventData model
- impl parsed event data in `parse_borrowing_event`

### CODE REVIEW

**Review by djeck1432 — CHANGES_REQUESTED** [review]
where is the last part?
In method [process_borrowing_event](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/handlers/loan_states/zklend/events.py#L281)
substitute fetching data by index Example: user= data[0] to data.user using serializer class

File: data_handler/handlers/loan_states/zklend/events.py [hunk 1]
```diff
@@ -180,10 +180,11 @@ def process_accumulators_sync_event(self, event: pd.Series) -> None:
     def process_deposit_event(self, event: pd.Series) -> None:
         # The order of the values in the `data` column is: `user`, `token`, `face_amount`.
         # Example: https://starkscan.co/event/0x036185142bb51e2c1f5bfdb1e6cef81f8ea87fd4d777990014249bf5435fd31b_3.
-        user = add_leading_zeros(event["data"][0])
-        token = add_leading_zeros(event["data"][1])
+        event_data = ZklendDataParser.parse_borrowing_event(event["data"])
+        user = add_leading_zeros(event_data.user)
```
Comment (djeck1432) [inline data_handler/handlers/loan_states/zklend/events.py:184]: move this logic `add_leading_zeros` to Model level not here

File: data_handler/handlers/loan_states/zklend/events.py [hunk 2]
```diff
@@ -180,10 +180,11 @@ def process_accumulators_sync_event(self, event: pd.Series) -> None:
...
+        token = add_leading_zeros(event_data.token)
```
Comment (djeck1432) [inline data_handler/handlers/loan_states/zklend/events.py:185]: move this logic `add_leading_zeros` to Model level not here

File: data_handler/handlers/loan_states/zklend/events.py [hunk 3]
```diff
@@ -180,10 +180,11 @@ def process_accumulators_sync_event(self, event: pd.Series) -> None:
...
 
-        face_amount = decimal.Decimal(str(int(event["data"][2], base=16)))
+        face_amount = decimal.Decimal(str(int(event_data.face_amount, base=16)))
```
Comment (djeck1432) [inline data_handler/handlers/loan_states/zklend/events.py:187]: move this logic `decimal.Decimal(str(int(event_data.face_amount, base=16)))` to Model level

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Please, fix merge conflict

**Comment by djeck1432** [comment]
@ikemHood Please, resolve conflicts

**Comment by ikemHood** [comment]
Done

**Pull request merged**